src/components/TodoList.tsx

```diff
@@ -2,11 +2,11 @@
 import TodoListItem from "./TodoListItem";
 import style from "./TodoList.module.css";
 
-const TodoList = (props) => {
+const TodoList = (props: { todoList: any; onRemoveTodo: any }) => {
```

Comment:
in your types.tsx declare a type called

```
   export interface TodoListProps {
        todoList : TtodoItemList
        onRemoveTodo: (id: string) => void
    }
```
then import it in this file

```suggestion
const TodoList = (props: TodoListProps) => {
```